nat-lab/tests/conftest.py

```diff
@@ -1,12 +1,25 @@
 import asyncio
 import pytest
-from helpers import SetupParameters
+import time
+from contextlib import AsyncExitStack
+from helpers import SetupParameters, setup_connections
+from interderp_cli import InterDerpClient
 from telio import AdapterType
 from typing import List, Tuple
 from utils.connection_util import ConnectionTag, LAN_ADDR_MAP
 from utils.router import IPStack
 from utils.vm import windows_vm_util, mac_vm_util
 
+DERP_SERVER_1_ADDR = "http://10.0.10.1:8765"
+DERP_SERVER_2_ADDR = "http://10.0.10.2:8765"
+DERP_SERVER_3_ADDR = "http://10.0.10.3:8765"
+DERP_SERVER_1_SECRET_KEY = "yBTYHj8yPlG9VtMYMwJSRHdzNdyAlVXGc6X2xJkjfHQ="
+DERP_SERVER_2_SECRET_KEY = "2NgALOCSKJcDxwr8MtA+6lYbf7b98KSdAROGoUwZ1V0="
+
+SETUP_CHECKS = [
+    ("setup_check_interderp", 10.0),
```

Comment:
can't we just add function here, instead of name? wouldn't need to use `globals()` later on  :thinking:

Reply:
I was having the impression, that it is not possible in python, but will try

Reply:
Done